src/amo/components/AddonReviewListItem/index.js

```diff
@@ -231,7 +231,12 @@ export class AddonReviewListItemBase extends React.Component<Props> {
         {reviewBody}
         <div className="AddonReviewListItem-byline">
           {review && !isReply ?
-            <Rating styleName="small" rating={review.rating} readOnly />
+            <Rating
+              styleSize="small"
+              review={review}
```

Comment:
It would be nice to add some test coverage to make sure that `Rating` gets passed the `review` property since that's an important part of the component interface and we don't have Flow. The test suite for this is old but you can maybe edit [a test like this](https://github.com/bobsilverberg/addons-frontend/blob/eae1d1ca2d834ebad45ff3d7f9dcacc157bf45d2/tests/unit/amo/components/TestAddonReview.js#L110).